.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Bug
+about: Something in the proxy is misbehaving
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
```

Comment:
Todo: get the reporter to tell us their homeserver address, or at least if they're self-hosted.

Reply:
Todo: tell them to open an issue on the client repo if they think this is a problem with EX rather than the proxy.

Reply:
Todo: get them to say what proxy version they're running.